mh/session.py

```diff
@@ -297,6 +299,21 @@ def try_transfer_city_leadership(self):
             return player
         return None
 
+    def try_transfer_circle_leadership(self):
+        if self.local_info['circle_id'] is None:
+            return None
```

Comment:
- Please add a `TODO` or `if not self.local_info['circle_id']`
 - `return None, None`

Reply:
Done